.github/workflows/release.yaml

```diff
@@ -0,0 +1,42 @@
+name: Create and publish a container image
+
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9 # v1.10.0 -- https://github.com/docker/login-action/releases/tag/v1.10.0  --- github recommends that we refer to the full hash when referencing external actions to reduce risk of a supply chain attack:  https://securitylab.github.com/research/github-actions-building-blocks/
```

Comment:
Maybe use `v1-0-g42d299f` as revision? It's a pain to update a bit though, as it requires cloning the repository to run `git describe --long --tags --always v1` or constructing the revision by hand using `<tag>-0-g<commit>`.